source/index.html.md

```diff
@@ -2001,7 +2002,7 @@ Field Name | Type | Description | Units
 **updated_at** | String | Timestamp at which the order was updated (can be null) | ISO 8601 timestamp
 **price** | String | The order's price | Minor
 **side** | String | The order side (buy, sell) | -
-**status** | String | The order's status (queued, open, partially filled, closed) | -
+**status** | String | The order's status (queued, open, partially filled, closed, completed, cancelled) | -
```

Comment:
Actually, an order can never be `closed` so we could remove this option

Reply:
Hm...OK, not sure what it was added in the first place, but let's remove it then...